projects/swimlane/ngx-ui/src/lib/components/dialog/dialog.component.html

```diff
@@ -1,20 +1,27 @@
 <div [class]="class" [class.ngx-dialog]="true" [style.zIndex]="zIndex">
   <div
     class="ngx-dialog-content {{ cssClass }}"
+    [attr.data-dialog-format]="format"
```

Comment:
Why use an attr here instead of a class?

Reply:
Because I didn't want to expose the `DialogFormat` enum on the template to use it like:

```html
[class.ngx-large-dialog]="format === DialogFormat.Large"
```

If this is a consistency issue you're seeing, I'm happy to change it to a class.